src/page-multi-facility/ScenarioSidebar.tsx

```diff
@@ -1,11 +1,21 @@
+import { saveScenario } from "../database";
 import PromoBoxWithButton from "../design-system/PromoBoxWithButton";
 import ToggleRow from "./ToggleRow";
+import { Scenario } from "./types";
+
+interface Props {
+  scenario?: Scenario | null;
+}
+
+const ScenarioSidebar: React.FC<Props> = (props) => {
+  // Typescript error that these properties do not exist on type Scenario
+  // Have I defined the type wrong?
+  const { name, dailyReports, dataSharing } = props.scenario;
```

Comment:
I'm still having issues here with the typescript error saying that `Scenario` type does not have these properties.